src/integrationTest/sam.test.ts

```diff
@@ -112,6 +112,15 @@ const scenarios: TestScenario[] = [
     //     language: 'python',
     //     dependencyManager: 'pip',
     // },
+    {
+        runtime: 'python3.10',
+        displayName: 'python 3.10 (ZIP)',
+        path: 'hello_world/app.py',
+        debugSessionType: 'python',
+        language: 'python',
+        dependencyManager: 'pip',
+        vscodeMinimum: '1.50.0',
```

Comment:
I see that only the "minimum" integ tests CI is failing. Might make sense to bump this (and the other one) to only run on 1.65 or so. We need to bump our minimum anyway.